controlpanel/api/auth0.py

```diff
@@ -629,6 +639,14 @@ def has_existed(self, user_id):
 
         return len(response["users"]) > 0
 
+    def remove_role(self, user_id, role_id):
+        response = self.remove_roles(user_id, [role_id])
+
+        if "error" in response:
+            raise Auth0Error("reset_mfa", response)
```

Comment:
Should this be something specific to removing the role? Looking at the [signature to `APIException` in the srouce code](https://github.com/encode/django-rest-framework/blob/master/rest_framework/exceptions.py#L108) it expects a message as the first arg, and a code as the second